dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/validator/relationship/RelationshipValidator.java

```diff
@@ -55,10 +54,9 @@ public RelationshipValidator()
                                 new ExistenceValidator(),
                                 new MandatoryFieldsValidator(),
                                 new MetaValidator(),
-                                new DataRelationsValidator(),
```

Comment:
Is the DataRelationsValidator unused now? I don't see the file removed in the diff.

Reply:
the `DataRelationsValidator` has been merged into `ConstraintEntityValidator`. Maybe it doesn't appear in the diff because I created it in this PR.